desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -125,12 +169,27 @@ void TabDiveInformation::updateData()
 	updateWhen();
 	ui->waterTemperatureText->setText(get_temperature_string(current_dive->watertemp, true));
 	ui->airTemperatureText->setText(get_temperature_string(current_dive->airtemp, true));
-	updateSalinity();
-
-	ui->atmPressType->setEditable(true);
```

Comment:
Why is this setEditable removed?

Reply:
Because it is set to FALSE at the start and because we do not want the dropdown list to be user-editable. Makes sense?

Reply:
But what did this code do to begin with? This is one of the problems with huge commits containing multiple changes combined into one, that there are no information on _WHY_ a certain chang was done.

Reply:
It sets up the dropdown list used for selecting the appropriate unit for specifying atmospheric pressure. The function initialises the values in the Information tab.

Reply:
You created the setEditable(true) , setItemText, setEditable(false) sance back in 1bdf00b2b472078a0b24f1c269782d822cb96e02 , and now you remove the setEditable(true) bit of it. Either leave it there, or remove both the setEditable(true) and setEditable(false) bits and write about why in the commit message.

Reply:
I removed the setEditable statements. Initially I had some problems with setItemText and I experimented with setEditable to get the setItemText to work. Clearly setEditable is not appropriate.

Reply:
But why is the setEditable(false) still there then, and why isn't this mentioned in the commit message?

Reply:
Can you still see it? I cannot.

Reply:
Now its gone.